File: Lesson 1/Example/One/book.py
class Book:
    """
    The attributes and behavors for all books go here
    """
    # constructor: code that is called automatically each time
    # a Book object is made
    def __init__(self, title, year_published, price_cad, fiction, version):
        self.title          = title
        self.year_published = year_published
        self.price_cad      = price_cad
        self.fiction        = fiction
        self.version        = version

File: Lesson 1/JavaBookDemo/src/com/DemoOne/Book.java
package com.DemoOne;

public class Book {
    // instance variables:
    private final String title;
    private final int yearPublished;
    private final boolean fiction;
    private final char edition;
    private double priceCad;

    // constructor
    Book(final String title, final int yearPublished, final double priceCad, final boolean fiction, final char edition)
    {
        this.title = title;
        this.yearPublished = yearPublished;
        this.priceCad = priceCad;
        this.fiction = fiction;
        this.edition = edition;
    }

    public String getTitle()
    {
        return title;
    }

}
